cmd/session-resume.go

```diff
@@ -38,14 +38,17 @@ var sessionResume = cli.Command{
   {{.HelpName}} - {{.Usage}}
 
 USAGE:
-  {{.HelpName}} [SESSION-ID/all]
+  {{.HelpName}} SESSION-ID
+
+SESSION-ID:
+  SESSION - Session can either be $SESSION-ID
```

Comment:
incomplete like @krisis pointed in previous instances

Reply:
Not incomplete `resume` is purposefully not supporting `all`

Reply:
@harshavardhana then the description shouldn't be `SESSION - Session can either be ...`, right?

Reply:
yes removing @krisis